pkg/apiserver/iam/im.go

```diff
@@ -160,6 +160,11 @@ func UpdateUser(req *restful.Request, resp *restful.Response) {
 		}
 	}
 
+	if usernameInHeader == user.Username {
+		// change cluster role by self is not permitted
```

Comment:
how about admin changes itself? Still not allowed? If so, it requires a different admin to change it?

Reply:
Yes, cluster role can only be modified by other admin or user manager.